api/src/main/java/org/openmrs/util/OpenmrsUtil.java

```diff
@@ -546,49 +548,41 @@ public static void applyLogLevels() {
 	public static void setupLogAppenders() {
 		Logger rootLogger = (Logger) LogManager.getRootLogger();
 		LoggerContext loggerContext = rootLogger.getContext();
-
+		
 		// stop the LoggerContext to reconfigure the scanning interval
 		loggerContext.stop();
 		// previously set in web.xml
 		loggerContext.getConfiguration().getWatchManager().setIntervalSeconds(60000);
 		loggerContext.start();
-
+		
 		RollingFileAppender fileAppender = null;
 		for (Map.Entry<String, Appender> appenderEntry : rootLogger.getAppenders().entrySet()) {
 			Appender appender = appenderEntry.getValue();
-			if (appender instanceof RollingFileAppender && OpenmrsConstants.LOG_OPENMRS_FILE_APPENDER
-				.equals(appender.getName())) {
+			if (appender instanceof RollingFileAppender
+			        && OpenmrsConstants.LOG_OPENMRS_FILE_APPENDER.equals(appender.getName())) {
 				fileAppender = (RollingFileAppender) appender;
 				break;
 			}
 		}
-
+		
 		if (fileAppender != null) {
 			rootLogger.removeAppender(fileAppender);
 		}
-
+		
 		String logLayout = Context.getAdministrationService().getGlobalProperty(OpenmrsConstants.GP_LOG_LAYOUT,
-			"%p - %C{1}.%M(%L) |%d{ISO8601}| %m%n");
-		PatternLayout patternLayout = PatternLayout.newBuilder()
-			.withPattern(logLayout)
-			.build();
-
+		    "%p - %C{1}.%M(%L) |%d{ISO8601}| %m%n");
+		PatternLayout patternLayout = PatternLayout.newBuilder().withPattern(logLayout).build();
+		
 		String logLocation = OpenmrsUtil.getOpenmrsLogLocation();
 		String logPattern = logLocation.replace(".log", ".%i.log");
-		fileAppender = RollingFileAppender.newBuilder()
```

Comment:
Please remove these formatting changes. Try to only add the changes you want to make regarding the Path/Paths into the PR.